app/views/red_line_boundary_change_validation_requests/_form.html.erb

```diff
@@ -32,7 +32,9 @@
                                         size: 's',
                                         class: 'govuk-label govuk-label--s govuk-!-padding-bottom-4' },
                                class: "govuk-visually-hidden" %>
-      <%= render "shared/location_map", locals: { geojson: @planning_application.boundary_geojson, geojson_field: 'red-line-boundary-change-validation-request-new-geojson-field' } %>
+
+      <% geojson = @red_line_boundary_change_validation_request.new_geojson.present? ? @red_line_boundary_change_validation_request.new_geojson : @planning_application.boundary_geojson %>
```

Comment:
I wonder if it's worth extracting this into a `#geojson` view helper method? Or even setting `@geojson` in the controller and using that?